### DISCUSSION TITLE
Microsoft account authorisation blocked with "admin approval required"

### DISCUSSION THREAD

**Discussion by phw198** [phw198/OutlookGoogleCalendarSync#1889] [Outlook Online]
The following screen may well be encountered if your corporate IT have blocked third party applications from accessing their accounts. 

If this affected you, [please take a second to respond to the poll](https://github.com/phw198/OutlookGoogleCalendarSync/discussions/1890), so I can get a feel for how many users are impacted.

Below are some examples of what the block screen may look like:-

<img src="https://github.com/phw198/OutlookGoogleCalendarSync/assets/11843015/3eae2b4d-649b-4e2c-ab0d-1a0b3143bcfd" width="50%"><img src="https://github.com/phw198/OutlookGoogleCalendarSync/assets/26889446/a071bdc7-5c44-4cc3-b90c-8e5037117112" width="50%">

**Comment by arshnawaj** [comment]
Yup I got the same issue
